crates/consensus/src/signed.rs

```diff
@@ -263,6 +268,187 @@ impl<'a, T: SignableTransaction<Signature> + arbitrary::Arbitrary<'a>> arbitrary
     }
 }
 
+impl<T> Typed2718 for Signed<T>
```

Comment:
ah yeah, we can easily add these trait impls, because Signed isn't deref so this shouldn't conflict